repl/src/test/scala/ammonite/repl/ScriptTests.scala

```diff
@@ -271,7 +271,7 @@ object ScriptTests extends TestSuite{
       }
       'caching{
 
-        def createTestInterp(storage: Storage) = new Interpreter(
+        def createTestInterp(storage: Storage, predefS: String = "") = new Interpreter(
```

Comment:
Oh ok that's fine then, can we remove the default argument and pass in the `predefS` manually for those cases?

Also, let's just call it `predef` instead of `predefS`